language/resource-api/README.md

```diff
@@ -214,15 +250,15 @@ In many cases, the resource type follows the conventional patterns of puppet, an
 
 `SimpleProvider` requires that your type follows some common conventions:
 
-* `name` is the name of your namevar attribute
+* the single namevar attribute is called `name` or the type has more than one namevar
```

Comment:
This doesn't make sense to me as a sentence. Perhaps:

A single namevar attribute called `name`; or, `names` if the type has multiple namevar attributes, which will receive a hash.

Reply:
changed it back to "if the type has a single namevar, it must be called `name`"

looking at the code, I *think* we can relax that, too. but that's a conversation for another day.

Reply:
👍